cmd/korg/korg.go

```diff
@@ -146,9 +150,35 @@ func main() {
 		},
 	}
 
-	addCmd.Flags().StringSliceVar(&o.Orgs, "org", []string{}, "orgs to add the user to")
+	auditCmd := &cobra.Command{
+		Use:   "audit",
+		Short: "Audit GitHub org members",
+		Long:  auditHelpText,
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			if invalidOrgs := findInvalidOrgs(o.Orgs); len(invalidOrgs) > 0 {
+				return fmt.Errorf("specified invalid orgs: %s", strings.Join(invalidOrgs, ", "))
+			}
+
+			// TODO: Check if exceptions file is of the right format, if defined
+
+			return nil
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return OrgAudit(o)
+		},
+	}
+
+	// flags
+	auditCmd.Flags().IntVar(&o.ActivityThreshold, "activity-threshold", 50, "minimum activity to be considered active. default: 50")
```

Comment:
Can we add validation for non-negativeness of `activity-threshold` in `PreRunE`? Or is that a valid input? If negative is passed, all users would be above threshold I think.